src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Lottery {
+    private final ColorSupplier cs = new ColorSupplier();
```

Comment:
proper naming is important - you don't save much time typing `cs` instead of `colorSupplier`, but it instantly makes your code way more readable for others. Remember, that coder spends way more time reading existing code than writing a new one